archinstall/lib/menu.py

```diff
@@ -0,0 +1,91 @@
+from .simple_menu import TerminalMenu
+
+
+class Menu(TerminalMenu):
+	def __init__(self, title, options, skip=True, multi=False, default_option=None, sort=True):
+		"""
+		Creates a new menu
+
+		:param title: Text that will be displayed above the menu
+		:type title: str
+
+		:param options: Options to be displayed in the menu to chose from;
+		if dict is specified then the keys of such will be used as options
+		:type options: list, dict
+
+		:param skip: Indicate if the selection is not mandatory and can be skipped
+		:type skip: bool
+
+		:param multi: Indicate if multiple options can be selected
+		:type multi: bool
+
+		:param default_option: The default option to be used in case the selection processes is skipped
+		:type default_option: str
+
+		:param sort: Indicate if the options should be sorted alphabetically before displaying
+		:type sort: bool
+		"""
+
+		if isinstance(options, dict):
+			options = list(options)
+
+		if sort:
+			options = sorted(options)
+
+		self.menu_options = options
+		self.skip = skip
+		self.default_option = default_option
+		self.multi = multi
+
+		menu_title = f'\n{title}\n\n'
+
+		if skip:
+			menu_title += "Use ESC to skip\n\n"
+
+		if default_option:
+			# if a default value was specified we move that one
+			# to the top of the list and mark it as default as well
+			default = f'{default_option} (default)'
+			self.menu_options = [default] + [o for o in self.menu_options if default_option != o]
+
+		cursor = "> "
+		main_menu_cursor_style = ("fg_red", "bold")
```

Comment:
Not a huge fan of this color scheme. Maybe we could go with something close to the Arch blue to match?